### ISSUE TITLE
Disable default entities

### ISSUE THREAD

**Issue by jokajak** [andweeb/ki#7]
There does not appear to be a way to disable loading default entities. For example, I'd like to replace the entity definition for f which is defaulted to Finder. Using the following it allows me to launch firefox but the cheatsheet lists both Finder and Firefox for the same key in the entity mode.
```lua
s = spoon.Ki
-- Create new Ki applications
local Application = s.Application
local GoogleChrome = Application:new("Google Chrome")
local Firefox = Application:new("Firefox")
local VSCode = Application:new("Visual Studio Code")
-- Define custom entity workflows
local entityEvents = {
    { nil, "g", GoogleChrome, { "Entities", "Google Chrome" }},
    { nil, "f", Firefox, { "Entities", "Firefox" }},
    { nil, "v", VSCode, { "Entities", "VSCode" }},
}
-- Use relevant application entities that can have multiple windows to be available in select mode
local selectEntityEvents = {
    { nil, "g", GoogleChrome, { "Entities", "Google Chrome" }},
    { nil, "f", Firefox, { "Entities", "Firefox" }},
    { nil, "v", VSCode, { "Entities", "VSCode" }},
}
s.workflowEvents = {
    defaultEntities = {},
    entity = entityEvents,
    select = selectEntityEvents,
}
s.defaultEntities = {}
s:start()

```

**Comment by andweeb** [comment]
Hi @jokajak, you were close!

The `Ki.defaultEntities` variable is there mostly for reference, to expose the underlying entities, so setting that to `{}` won't actually do anything (or it might throw errors when you try using the default shortcuts to use those entities).

If you intend to disable _all_ shortcuts, [`Ki.defaultWorkflowEvents`](https://andweeb.github.io/ki/html/Ki.html#defaultWorkflowEvents) may be what you're looking for. You can use `Ki.defaultWorkflowEvents.entity = {}` if you want to disable the default shortcuts for entities specifically.

If what you want is to override a single shortcut, you can simply set your custom entity events to `Ki.workflowEvents` (as you're already doing) and the `start` method will override them, which is why that Firefox shortcut is working for you.

The duplication you see in the cheatsheet is actually visual bug, thanks for reporting it! I'll see about putting out a fix for it when I can.

**Comment by andweeb** [comment]
Resolved in #9
